qoqo-iqm/src/backend.rs

```diff
@@ -46,18 +46,26 @@ impl BackendWrapper {
     ///
     /// Args:
     ///     input (Backend): The Python object that should be casted to a [roqoqo_iqm::Backend]
-    pub fn from_pyany(input: Py<PyAny>) -> PyResult<Backend> {
+    pub fn from_pyany(input: PyObject) -> PyResult<Backend> {
```

Comment:
Why has this type been changed? Should the function name be changed too?

Reply:
PyObject is just an alias for Py<PyAny>. I changed it because it sounds more descriptive of the type to me